### PR TITLE
Try for inline T&C links on small screens

### PR DESCRIPTION
**Pull request by jaw6**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

On small screens (below our xs breakpoint), links to terms & conditions on the onboarding screen break in a visually odd way:

179087472-eaf51440-e9f8-48da-8d3f-8ad8495ad34d

It seems that at a small enough screen dimension, the `label` container becomes a two-column layout, with links and text in separate columns. This feels like a bug in the grid layout -- it doesn't seem to be something we've _asked for_ -- but I might be missing something?

Anyway, after:

Screen Shot 2022-07-22 at 15 35 56

## Related Tickets & Documents

- Closes https://github.com/forem/forem-internal-eng/issues/487

## QA Instructions, Screenshots, Recordings

To replicate the issue on a desktop browser, I needed to use device emulation mode -- otherwise, the browser's minimum width was wider than the width at which the bug would occur. Depending on browsers, odd wrapping happens somewhere around 400±25px wide.

### UI accessibility concerns?

The links were functional all along, just wrapping in an odd way. 

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: surface-level visual change
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [x] This change does not need to be communicated, and this is why not: surface-level visual change

## [optional] Are there any post deployment tasks we need to perform?

Nope, just a visual tweak.

## [optional] What gif best describes this PR or how it makes you feel?

giphy

### CODE REVIEW

File: app/assets/stylesheets/preact/onboarding-modal.scss [hunk 1]
```diff
@@ -644,6 +644,10 @@ $onboarding-user-selected-hover: rgba(71, 85, 235, 0.2);
         align-items: center;
       }
 
+      @media screen and (max-width: $breakpoint-xs) {
```
Comment (aitchiss) [inline app/assets/stylesheets/preact/onboarding-modal.scss:647]: I think this is definitely looking much better, but it's bugging me that the alignment between the checkboxes and the text is a little off and we have quite a big gap between the lines when it wraps 🤔 

I had an experiment and if we remove this media query, and the one above on line 643, and instead add:

```
flex-wrap: wrap;
align-items: center;
```

to the main CSS, I think we have something that works for all screen sizes:

https://user-images.githubusercontent.com/20773163/181005111-2c37fdfc-ad76-4dc6-be98-4e7bba7e0d9d.mp4

What do you think?

Comment (jaw6) [reply app/assets/stylesheets/preact/onboarding-modal.scss]: Fantastic! :+1:

**Review by fdocr — APPROVED** [review]
Tried this out locally and IMO it does look better. Thanks @jaw6!

**Review by aitchiss — APPROVED** [review]
So much better! ✨

**Pull request merged**